cypress/locators/radioButton/index.js

```diff
@@ -1,17 +1,16 @@
 import {
-  RADIO_BUTTON,
   RADIO_BUTTON_COMPONENT,
   RADIO_BUTTON_GROUP_COMPONENT,
   RADIO_BUTTON_FIELDSET_GROUP,
 } from './locators';
+import { FIELD_HELP_PREVIEW } from '../locators';
 
 // component preview locators
-export const radioButtonComponent = () => cy.iFrame(RADIO_BUTTON_COMPONENT);
-export const radioButtonByPosition = position => cy.iFrame(RADIO_BUTTON).eq(position);
-
-// no Iframe locators
 export const radioButtonFieldset = () => cy.get(RADIO_BUTTON_FIELDSET_GROUP).children();
 export const radioButtonLegend = () => cy.get(RADIO_BUTTON_FIELDSET_GROUP).find('legend').parent();
 export const radioButtonGroupNoIframe = () => cy.get(RADIO_BUTTON_GROUP_COMPONENT);
 export const radioButtonComponentByPositionNoIframe = position => cy.get(RADIO_BUTTON_COMPONENT)
   .eq(position);
+export const radioButtonComponentNoIframe = () => cy.get(RADIO_BUTTON_COMPONENT);
+export const radioButtonInputByPosition = position => radioButtonComponentByPositionNoIframe(position).find('input');
+export const radioButtonFieldHelp = position => radioButtonComponentByPositionNoIframe(position).find(FIELD_HELP_PREVIEW);
```

Comment:
```suggestion
export const radioButtonFieldHelpByPositionNoIFrame = position => radioButtonComponentByPositionNoIframe(position).find(FIELD_HELP_PREVIEW);
```

Reply:
as we discussed if we using it in `noIframe` the name is e.g `radioButtonFieldHelp`.`